python/ray/_private/worker.py

```diff
@@ -2489,6 +2498,20 @@ def show_in_dashboard(message: str, key: str = "", dtype: str = "text"):
 blocking_get_inside_async_warned = False
 
 
+def _end_read_channel(object_refs):
```

Comment:
Shall we move these methods to the dag/channel subpackage?

Reply:
+1